scripts/future-fetch.py

```diff
@@ -45,6 +45,9 @@ class Entry(object):
     witnesses_str = ''
     location = ''
 
+    # strip 4 byte unicode chars plus zero width space
+    re_pattern = re.compile(u'(?:[^\u0000-\uD7FF\uE000-\uFFFF]|[\u200B])', re.UNICODE)
```

Comment:
I think any character that won't transcode to CP1252 will cause an issue; it looks like we either need to put back the ISO-8859-1-and-charrefs encoding removed in https://github.com/mysociety/theyworkforyou/commit/87768f573b3351ccc77bbe3d16e5351db5b12fb2 or actually switch the DB column character sets as originally suggested in #1197.